packages/garbo/src/tasks/freeGiantSandworm.ts

```diff
@@ -130,6 +134,17 @@ function nonSandwormTask(
 
 const sandwormMacro = () => Macro.trySingAlong().tryHaveSkill($skill`Otoscope`);
 
+function safeSweatBulletCasts(): number {
+  const mainstat = myPrimestat();
+  return mainstat === $stat`Moxie` && myLevel() >= 23
+    ? BloodCubicZirconia.availableCasts($skill`BCZ: Sweat Bullets`, 629) // If we're level 26, we don't want to go under that
+    : mainstat === $stat`Moxie` && myLevel() >= 20
+      ? BloodCubicZirconia.availableCasts($skill`BCZ: Sweat Bullets`, 365) // If we're level 20, we don't want to go under that
+      : mainstat === $stat`Moxie`
+        ? BloodCubicZirconia.availableCasts($skill`BCZ: Sweat Bullets`, 148) // If we're a moxie class, we don't want to go under level 13
+        : BloodCubicZirconia.availableCasts($skill`BCZ: Sweat Bullets`, 100);
```

Comment:
This is kinda hard to read. It feels like it would be easier to code using a series of nested conditionals:
```suggestion
  if (myPrimestat() === $stat`Moxie`) {
    if (myLevel() >= 23) return BloodCubicZirconia.availableCasts($skill`BCZ: Sweat Bullets`, 629);
    if (myLevel() >= 20) return BloodCubicZirconia.availableCastss($skill`BCZ: Sweat Bullets`, 365);
    return BloodCubicZirconia.availableCast($skill`BCZ: Sweat Bullets`, 148);
  }
  return BloodCubicZirconia.availableCasts($skill`BCZ: Sweat Bullets`,  100);
```
(also, as discussed in discord, 23 is wrong)

(also, I'm not sure we _do_ want to cast down to 100; it's really nice to be able to equip stuff like the Crumpled Felt Fedora)